stdlib/stdlib-external/ascutils/dataStructures.ts

```diff
@@ -103,6 +123,12 @@ export function rawInt64Ptr(wrappedInt64: u32): u32 {
   return wrappedInt64 + 8
 }
 
+// @ts-ignore: decorator
+@inline
+export function loadI64(xptr: u32): i64 {
```

Comment:
I know you just moved them, but out of the context of their original files, the names are maybe a tad confusing. Can you change them to match the names of the Grain types (and their corresponding allocation functions), i.e. `Int32`, `Int64`, `Float32`, `Float64`?

Reply:
No worries. Fixed